webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
```

Comment:
Also let's add `resolve` section. It allows to avoid extension in file names: https://webpack.js.org/configuration/resolve/#resolve-extensions. We need `resolve: { extensions }`